src/content_main.js

```diff
@@ -171,42 +163,134 @@ async function storeRecentTimeSpent(recentTimeSpent) {
 }
 
 /***********************************************************
- * Validate Payment Pointer
+ * Track and Validate Payment Pointer
  ***********************************************************/
 
 /**
- * Check for a monetization meta tag on the website and verify that
- * the payment pointer is valid (resolves to a valid SPSP endpoint).
- * 
- * TODO: use enum to indicate no meta tag, meta tag + valid endpoint,
- * meta tag + invalid endpoint.
- * 
- * @return {Promise<{ isPaymentPointerValid: boolean, paymentPointer:string}>} 
- * isPaymentPointerValid is true if both monetization is present and the payment endpoint is valid.
- * paymentPointer is the paymentPointer if it is found in the monetization meta tag, otherwise null.
+ * getPaymentPointerFromPage gets the payment pointer as a string from the monetization meta tag
+ * on the current page. If there is no monetization meta tag on the page, null is returned.
```

Comment:
Don't need function name in doc description (remove `getPaymentPointerFromPage` and gets --> Gets`